examples/00_basic/server.py

```diff
@@ -1,102 +1,172 @@
-# 
-# Server
-# 
-# Example 0: Global and User Text Editor
-# ======================================
-#
-# This server demonstrates the access control of two endpoints:
-#  - /text/global.txt
-#  - /text/user/<username>.txt
-#
-# All registered users can read and write to the /text/global.txt, however
-# only a user can read and write to their own /text/user/<username>.txt file.
-#
-# The user management endpoints are implemented by the flask_restful_auth
-# package. By default, they include:
-# - /user/signup
-# - /user/login
-# - /user/logout
-
-from flask import Flask
+import os
+from flask import Flask, jsonify, request, make_response
 from flask_sqlalchemy import SQLAlchemy
-from flask_restful_auth import RestfulAuth, login_required
-from flask import request
-from flask_login import UserMixin, login_required
-
-def create_app():
-    """ Flask application factory. """
-
-    app = Flask(__name__)
-    app.config['SECRET_KEY'] = 'my little secret'
-    app.config['SECRET'] = 'my little secret'
-
-    # Initialise Flask-SQLAlchemy
-    db = SQLAlchemy(app)
-
-    app.db = db
-
-    # Create the user class.
-    class User(db.Model, UserMixin):
-        ''' Setting up User'''
-        id = db.Column(db.String(36), primary_key=True)
-        is_authenticated = db.Column(db.Boolean, nullable=False)
-        is_active = db.Column(db.Boolean, nullable=False)
-        # User vereification
-        username = db.Column(db.String(60), nullable=False, unique=True)
-        password = db.Column(db.String(300), nullable=False)
-        token = db.Column(db.String(500), nullable=True)
-
-    # Create tables
-    db.create_all()
-
-    import uuid
-    from passlib.hash import sha256_crypt
-    new_user = User(
-        id=str(uuid.uuid4()),
-        is_active=True,
-        is_authenticated=False,
-        password=sha256_crypt.hash('password'),
-        username='u1',
-    )
+from functools import wraps
+from passlib.hash import sha256_crypt as sc
+import uuid
+import secrets
+import jwt
+import datetime
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = os.environ.get('Secret')
+secret_key_2 = os.environ.get('Secret_2')
+app.config['SQLALCHEMY _DATABASE_URI'] = 'something'
```

Comment:
This should be set to memory: `'sqlite://'`. 
https://docs.sqlalchemy.org/en/14/core/engines.html#sqlite

Reply:
oh okay, Actually  I haven't tested the code yet.
I'll correct it.